### ISSUE TITLE
Not changing clothes

### ISSUE THREAD

**Issue by OKAllu** [olandra/CBSafe-Simulation#18]
Produced with 101:
- After finishing surface rescue, diving gear stays on even after issuing next command 
--> Extinguishing fire with diving gear on so no water either, since divers don't have waterhose in hand

**Comment by olandra** [comment]
This hasn't been implemented due to the lack of time earlier. It was deemed unimportant due to the 20 minute timer per session.

This is related to https://trello.com/c/BlonlS7y/46-idle-in-swimming-suit

**Comment by olandra** [comment]
Changing clothes before starting a new task now works with changeset 686. Implementation for the timer comes later if there is time.
